proofread docs: for_local.md (#1565)
juicedata/juicefs: docs/en/getting-started/for_local.md

# JuiceFS Quick Start Guide for Standalone Mode

The JuiceFS file system is driven by both ["Object Storage"](../reference/how_to_setup_object_storage.md) and ["Database"](../reference/how_to_setup_metadata_engine.md). In addition to object storage, it also supports to use local disk, WebDAV and HDFS, etc., as underlying storage. Therefore, you can create a standalone file system using local disks and SQLite database to get a quick overview of how JuiceFS works.

## Install Client

For details, please refer to [Installation & Upgrade](installation.md).

A help message will return after executing `juicefs` in terminal once the JuiceFS client is successfully installed regardless of operating system.

## Creating a File System

### Basic Concept

> JuiceFS client provides [`format`](../reference/command_reference.md#juicefs-format) command to create a file system. The general format is shown below.

Author: ```suggestion
JuiceFS client provides a command [`format`](../reference/command_reference.md#juicefs-format) to create a file system as follows.
```

> JuiceFS client provides [`format`](../reference/command_reference.md#juicefs-format) command to create a file system. The general format is shown below.

Author: ```suggestion
JuiceFS client provides a command [`format`](../reference/command_reference.md#juicefs-format) to create a file system as follows.
```

> JuiceFS client provides [`format`](../reference/command_reference.md#juicefs-format) command to create a file system. The general format is shown below.

Author: ```suggestion
JuiceFS client provides a command [`format`](../reference/command_reference.md#juicefs-format) to create a file system as follows.
```

2021/12/14 18:26:37.689683 juicefs[40362] <INFO>: Volume is formatted as {Name:myjfs UUID:d5bdf7ea-472c-4640-98a6-6f56aea13982 Storage:file Bucket:/Users/herald/.juicefs/local/ AccessKey: SecretKey: BlockSize:4096 Compression:none Shards:0 Partitions:0 Capacity:0 Inodes:0 EncryptKey:}
```

> As you can see from the output, the file system uses SQLite as the metadata storage engine and the database file is located in the current directory with the file name `myjfs.db`, which stores all the information of the `myjfs` file system. The information has been constructed into a complete table structure, which will be used to store all the meta information of the data.

Author: ```suggestion
As you can see from the output, the file system uses SQLite as the metadata storage engine. The database file is located in the current directory with the file name `myjfs.db`, which creates a table to store all the metadata of the file system `myjfs`.
```

    sqlite3://myjfs.db myjfs
```

> In the above command, the database and file system names remain the same, but the following information related to object storage needs to be added.

Reviewer: ```suggestion
The command above creates a file system with the object storage options provided, using the same database name and file system name.
```
原文这里也比较绕，感觉这样会不会更清晰一些？

Author: ```
The command above creates a file system using the same database name and file system name with the object storage options provided.
```
这样会不会更好一点？

- `--storage`: Used to set the storage type, e.g. oss, s3, etc.
- `--bucket`: Used to set the Endpoint address of the object storage.
- `--access-key`: Used to set the Object Storage Access Key ID.
- `--secret-key`: Used to set the Object Storage Access Key Secret.

:::note
Please replace the information in the above command with your own object storage information.
:::

You can mount the file system once it is created.

```shell
juicefs mount sqlite3://myjfs.db mnt
```

> As you can see, the mount command is exactly the same as using the local storage because JuiceFS has already written the information about the object storage to the `myjfs.db` database, so there is no need to provide it again when mounting.

Reviewer: ```suggestion
As you can see, the mount command is exactly the same as using the local storage because JuiceFS has already written the metadata about the object storage to the `myjfs.db` database, so there is no need to provide it again when mounting.
```

> Compared to local disks, combining SQLite with object storage has a higher utility value. From an application perspective, this approach is equivalent to plugging an object storage with almost unlimited capacity into your local computer, allowing you to use cloud storage as a local disk.

Reviewer: ```suggestion
Compared with using local disks, the combination of SQLite and object storage is more practical. From an application perspective, this approach is equivalent to plugging an object storage with almost unlimited capacity into your local computer, allowing you to use cloud storage as a local disk.
```
词典里 utility 的释义是
>the state of being useful, profitable, or beneficial.

暗含了 value 的意思，感觉 utility value 有点不 native
改成 more practical 会不会好点呢

> Further, all the data of the file system is stored in the cloud-based object storage, so the `myjfs.db` database can be copied to other computers where JuiceFS clients are installed for mounting, reading and writing. That is, any computer that can read the database which stores the metadata can mount to and read/write the file system.

Reviewer: ```suggestion
Further, all the data of the file system is stored in the cloud-based object storage, so the `myjfs.db` database can be copied to other computers where JuiceFS clients are installed for mounting, reading and writing. That is, any computer that can read the metadata database can mount and read/write the file system.
```
